src/Service/DataExporter.php

```diff
@@ -410,7 +523,57 @@ public function exportSystem($groupId = null, $splitIntoSubOwners = false)
             'systems',
             System::class,
             $entities,
-            $splitIntoSubOwners
+            $splitIntoSubOwners,
+            $onlyComments,
+            $withColor
         );
     }
+
+    /**
+     * Get a value corresponding to answer smiley selected.
+     *
+     * @param \App\Entity\Answer $answer
+     * @return int
+     */
+    private function getAnswerValue(Answer $answer)
+    {
+        if ($answer->getSmiley() == SmileyType::BLUE ||
```

Comment:
Why not use a `switch` statement (with fall-through) (as used in `getAnswerColor`)?

Reply:
Good point